application/src/test/java/uk/gov/hmcts/net/ccd/definition/store/rest/MultipleControllersEndpointIT.java

```diff
@@ -111,4 +117,44 @@ public void shouldReturnSingleWizardPageForTestComplexAddressBookCase() throws E
                 hasItem(Matchers.hasProperty("caseFieldId", containsString("ContectEmail"))))))
         );
     }
+
+    // To be @Nested - UserRoleController
+    @Test
+    public void shouldReturnUserRolesForDefinedRoles() throws Exception {
+        givenUserProfileReturnsSuccess();
```

Comment:
Why are we having this mocking here? Is it being used in the flow?

Reply:
removed this one